test/staff/get-one.test.ts

```diff
@@ -0,0 +1,91 @@
+import * as dotenv from 'dotenv'
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+dotenv.config()
+import { v0 } from '../../src/tillhub-js'
+import { initThInstance } from '../util'
+
+const legacyId = '4564'
+
+const mock = new MockAdapter(axios)
+afterEach(() => {
+  mock.reset()
+})
+
+const staffId = 'asdf5566'
+const staffMember = {
+  firstname: 'asdf',
+  lastname: 'asdf',
+  pin: 4567
+}
+
+describe('v0: Staff: can get one staff member', () => {
+  it("Tillhub's staff are instantiable", async () => {
+    if (process.env.SYSTEM_TEST !== 'true') {
+      mock.onPost('https://api.tillhub.com/api/v0/users/login').reply(function (config) {
+        return [
+          200,
+          {
+            token: '',
+            user: {
+              id: '123',
+              legacy_id: legacyId
+            }
+          }
+        ]
+      })
+
+      mock
+        .onGet(`https://api.tillhub.com/api/v0/staff/${legacyId}/${staffId}`)
+        .reply(function (config) {
+          return [
+            200,
+            {
+              count: 1,
+              results: [staffMember]
+            }
+          ]
+        })
+    }
+
+    const th = await initThInstance()
+
+    const Staff = th.staff()
+
+    expect(Staff).toBeInstanceOf(v0.Staff)
+
+    const { data } = await Staff.getOne(staffId)
+
+    expect(data).toMatchObject(staffMember)
+  })
+
+  it('rejects on status codes that are not 200', async () => {
+    if (process.env.SYSTEM_TEST !== 'true') {
+      mock.onPost('https://api.tillhub.com/api/v0/users/login').reply(function (config) {
+        return [
+          200,
+          {
+            token: '',
+            user: {
+              id: '123',
+              legacy_id: legacyId
+            }
+          }
+        ]
+      })
+
+      mock
+        .onGet(`https://api.tillhub.com/api/v0/staff/${legacyId}/staffId`)
```

Comment:
shouldn't `staffId` be templated, or is this intentional here?

Reply:
A total mistake. Good catch!